docs/Contributors_Guide/add_use_case.rst

```diff
@@ -1066,7 +1075,7 @@ MET Tool Wrapper Use Cases::
 
 Once you have verified the correct directories are set, copy the files::
 
-    cp -r $from_directory/* $to_directory/
+    cp -r $from_directory/ $to_directory/
```

Comment:
```suggestion
    cp -r $from_directory $to_directory/
```
I'm not 100% sure what would happen if the ending slash is included, but I know that without it, the entire directory named ${METPLUS_USE_CASE_NAME} would be put into the $to_directory. Including a slash may copy all of the files in the directory without the actual directory.